casper/src/main/scala/coop/rchain/casper/util/rholang/InterpreterUtil.scala

```diff
@@ -208,8 +208,17 @@ object InterpreterUtil {
       case Seq((_, parentStateHash)) =>
         parentStateHash.pure
 
-      case (_, initStateHash) +: _ =>
-        replayIntoMergeBlock(parents, dag, runtimeManager, initStateHash)
```

Comment:
Is this function (replayIntoMergeBlock) used anywhere else?

Reply:
https://rchain.atlassian.net/browse/RCHAIN-4016

Reply:
@tgrospic no, this is the only place where it is called.
Ticket created https://rchain.atlassian.net/browse/RCHAIN-4016.